symspellpy/symspellpy.py

```diff
@@ -315,6 +315,26 @@ def load_bigram_dictionary(
                 infile, term_index, count_index, separator
             )
 
+    def load_json(self, corpus: Dict[str, int]) -> None:
+        """Loads dictionary data from a JSON object.
+
+        Args:
+            corpus: A dictionary where keys are words and values are their frequencies.
+        """
+        self._words = corpus
```

Comment:
This bypasses [`create_dictionary_entry`](https://github.com/mammothb/symspellpy/blob/d4645ec461ce6c7e9628fe09e116d9911b55e2c5/symspellpy/symspellpy.py#L1137) which contains logic to handle only adding to `_words` if it reaches above the `count_threshold`.

`create_dictionary_entry` should be used so that the behavior of loading a dictionary from `.txt` or `.json` file is consistent